src/key.rs

```diff
@@ -726,6 +710,127 @@ pub fn import(
     }
 }
 
+/// The key data that makes up an RSA key.
+#[cfg(feature = "untested")]
+pub struct RsaKeyData {
+    /// The secret prime `p`.
+    p: Zeroizing<Vec<u8>>,
```

Comment:
You can use `crate::Buffer` for this (it's a type alias)

Reply:
Oh that's Handy, didn't notice that before.